README.md

```diff
@@ -1,4 +1,4 @@
-[![Dart](https://github.com/dart-lang/intl_translation/actions/workflows/build.yaml/badge.svg)](https://github.com/dart-lang/intl_translation/actions/workflows/build.yaml)
+[![Dart](https://github.com/dart-lang/intl_translation/actions/workflows/build.yaml/badge.svg?branch=master)](https://github.com/dart-lang/intl_translation/actions/workflows/build.yaml)
```

Comment:
Actually, I believe that the badge will default to reporting the build status from the default branch for the repo - you don't need to specify `master` explicitly.

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge